v2/datastream-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/DataStreamToSpannerSessionIT.java

```diff
@@ -171,7 +171,7 @@ public void migrationTestWithRenameAndDropColumn() {
   }
 
   @Test
-  public void migrationTestWithSyntheticPK() {
+  public void migrationTestWithSyntheticPKAndExtraColumn() {
     // Construct a ChainedConditionCheck with 2 stages.
     // 1. Send initial wave of events
     // 2. Wait on Spanner to have events
```

Comment:
We should also assert the contents of extra column in assertBooksBackfillContents method.

Reply:
As discussed offline, the `extraCol1` is null and asserting it seems to break `hasRecordsUnorderedCaseInsensitiveColumns` method. Going ahead with the current approach.